cve_bin_tool/checkers/sqlite.py

```diff
@@ -69,7 +69,7 @@ def guess_contains(lines, version_map):
     for line in lines:
         for signature in signatures:
             if signature in line:
-                return 1
+                return "1"
```

Comment:
As discussed, this shouldn't be needed because it's meant to be a true/false, and it's being changed in the work being done to convert checkers to a new base class anyhow.  You can remove this fix from the PR.